deploy/ocm-agent-custom-resource/config.yaml

```diff
@@ -0,0 +1,7 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  resourceApplyMode: Upsert
```

Comment:
I'd prefer this as `Sync` myself, that way if we ever needed to suddenly withdraw OA from all clusters we have that ability. Not sure that having it as `Upsert` is really giving us any advantages.

Reply:
I am having same thought and modified it to Sync in my last commit